bashlex/parser.py

```diff
@@ -638,8 +641,10 @@ def parse(s, strictmode=True, expansionlimit=None, convertpos=False):
 
     expansionlimit is used to limit the amount of recursive parsing done due to
     command substitutions found during word expansion.
+
+    when proceedonerror set, the parser will return AST nodes for unimplemented features, etc.
```

Comment:
when proceedonerror is set

maybe also add at the end "rather than throwing a NotImplementedError"